src/main/java/net/minecraftforge/event/entity/player/PlayerInteractEvent.java

```diff
@@ -326,6 +327,39 @@ public LeftClickEmpty(EntityPlayer player, @Nonnull ItemStack stack)
     }
 
     /**
+     * This event is fired on the client side when a player middle clicks while aiming on a block or entity.
+     * The server is not aware of when the client left clicks empty space, you will need to tell the server yourself.
+     * You can change the Result ItemStack here or set it to {@link ItemStack#EMPTY} or cancel the event to prevent further updating.
```

Comment:
`prevent further processing` -> `return nothing from the pick` (or something similar)